build.gradle

```diff
@@ -23,28 +23,25 @@ repositories {
 
 dependencies {
     // Spring Boot
-    implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    developmentOnly 'org.springframework.boot:spring-boot-devtools'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-webflux'
+    implementation 'org.springframework.boot:spring-boot-starter-websocket'
     annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
 
+    // MySQL
+    runtimeOnly 'mysql:mysql-connector-java'
+
     // Lombok
     compileOnly 'org.projectlombok:lombok'
     annotationProcessor 'org.projectlombok:lombok'
 
     // Spring API docs
     implementation 'org.springdoc:springdoc-openapi-ui:1.5.6'
-    implementation 'org.springframework.boot:spring-boot-starter-webflux'
 
     // Tests
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc'
-
-    // Google Guava
-    implementation 'com.google.guava:guava:30.1.1-jre'
-
-    // Web Socket
-    implementation 'org.springframework.boot:spring-boot-starter-websocket'
```

Comment:
- Removed unused dependencies
- Removed mongo dependency
- Added dependency for mysql and spring data jpa